src/components/graph/ExerciseGraphEdge.ts

```diff
@@ -0,0 +1,11 @@
+import { ExerciseGraphNode } from './ExerciseGraphNode'
+
+export class ExerciseGraphEdge {
+  public from: ExerciseGraphNode
+  public to: ExerciseGraphNode
+
+  constructor(from: ExerciseGraphNode, to: ExerciseGraphNode) {
+    this.from = from
+    this.to = to
+  }
+}
```

Comment:
So, here is a fun trick:

```typescript
export class ExerciseGraphEdge {
  constructor(public from: ExerciseGraphNode, public to: ExerciseGraphNode) {
  }
}
```

This will make both `constructor` arguments properties.

I would also consider making it immutable:

```typescript
export class ExerciseGraphEdge {
  constructor(public readonly from: ExerciseGraphNode, public readonly to: ExerciseGraphNode) {
  }
}
```